addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,97 @@
+# iCloud Binding
+
+The Apple [iCloud](https://www.apple.com/icloud/) is used to retrieve data such as the battery level or current location of one or multiple Apple devices connected to an iCloud account.
+Updates are quick and accurate without significant battery time impact.
+The Binding also offers access to the "Find My iPhone" function.
+
+An Apple account is required. Two factor authentication is supported.
+
+Please note: Application specific passwords are not supported.
+You´ll need to provide the account password to the binding.
+
+## Supported Things
+
+The following devices are known to work with this Binding:
+
+* iPhone 6s
+* iPhone 5c
+* iPhone 7
+* Apple Watch Series 2
+* iPad Air 2/2017/Pro
+* MacBook Pro
+
+Other devices connected to your iCloud account should work as well.
```

Comment:
I additionally have iPhone 4s, iPhone SE, MacBook Air 11", MacBook Pro 15", iPad 2, etc. and all turn up.

Is there any reason to believe that some device WON'T work? I don't think it makes sense to list the full Apple product portfolio here, so I would suggest to remove this list and instead only add a notice if we know that something does NOT work.

Reply:
My apple TV does not show ;-)

Reply:
Done.